include/swift/AST/DiagnosticsSema.def

```diff
@@ -562,6 +562,9 @@ ERROR(cannot_return_value_from_void_func,none,
 
 ERROR(sema_no_import,Fatal,
       "no such module '%0'", (StringRef))
+NOTE(sema_no_import_arch,Fatal,
```

Comment:
This part isn't going to work, unfortunately. Notes always have to be attached to an error or warning, but one hasn't been emitted yet in this case. You'll have to make this an error on its own.